modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/type/runtime/ConfigurationWrapper.java

```diff
@@ -23,16 +23,21 @@
  */
 class ConfigurationWrapper extends ComponentWrapper implements ConfigurationElement, ParameterizableTypeElement {
 
+  private final Set<ExtensionParameter> parameters;
+
   ConfigurationWrapper(Class aClass) {
     super(aClass);
+    this.parameters = Stream.concat(getAnnotatedFields(Parameter.class).stream(),
+                                    getAnnotatedFields(ParameterGroup.class).stream())
+        .distinct()
```

Comment:
we need a syntax validator that checks for usage of @Parameter along with @ParameterGroup in the same field, in the mean time we need to avoid the duplication if someone uses both annotations in the same param